packages/nodejs/package.json

```diff
@@ -32,25 +32,25 @@
     "es6-promise": "^4.2.6",
     "ethereumjs-tx": "^1.3.7",
     "ethjs": "^0.4.0",
+    "grpc": "^1.24.6",
     "ipfs-http-client": "^30.1.1",
     "lodash": "latest",
     "singularitynet-platform-contracts": "^0.3.4",
     "singularitynet-token-contracts": "^2.0.1",
-    "web3": "^1.3.0",
-    "winston": "^3.2.1"
+    "web3": "^1.3.4",
+    "winston": "^3.3.3"
   },
   "peerDependencies": {
-    "google-protobuf": "^3.8.0",
-    "grpc": "^1.21.1"
+    "google-protobuf": "^3.8.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.3.4",
-    "@babel/node": "^7.2.2",
-    "@babel/preset-env": "^7.3.4",
-    "@babel/register": "^7.0.0",
-    "google-protobuf": "^3.8.0",
-    "rimraf": "^2.6.3",
-    "verdaccio": "^4.8.1"
+    "@babel/cli": "^7.13.10",
```

Comment:
are all these recommended upgrades ?

Reply:
Added @babel/cli as a development dependency that is required to debug SDK locally or we need to explicitly add.